modules/govuk/manifests/apps/service_manual_frontend.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: govuk::apps::service_manual_frontend
+#
+# Service Manual Frontend is an app to serve content pages for the service manual from the content store
+#
+# === Parameters
+#
+# [*vhost*]
+#   Virtual host used by the application.
+#   Default: 'service-manual-frontend'
+#
+# [*port*]
+#   What port should the app run on?
+#
+class govuk::apps::service_manual_frontend(
+  $vhost = 'service-manual-frontend',
+  $port = 3122,
+) {
+  Govuk::App::Envvar {
+    app => 'service-manual-frontend',
+  }
+
+  govuk::app { 'service-manual-frontend':
+    app_type              => 'rack',
+    port                  => $port,
+    vhost_ssl_only        => true,
+    health_check_path     => '/healthcheck',
+    legacy_logging        => false,
+    asset_pipeline        => true,
+    asset_pipeline_prefix => 'service-manual-frontend',
+    vhost                 => $vhost,
+  }
+}
```

Comment:
Actually, https://github.com/alphagov/govuk-puppet/pull/3884 is a good example of adding `ENV` vars to an existing application definition.

You might also want to add a [feature flag](https://github.com/alphagov/govuk-puppet/commit/75c0b78f822fa939c7b9ea15cc799349f7c669b0) so the app can be enabled in integration only, while setting the app up.